backend/src/main/resources/db/migration/V1_0_12__Modify_Confirmed_Default_True.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE volunteers_proposals ALTER COLUMN confirmed SET DEFAULT true;
```

Comment:
You should not create 3 files in the same PR whihc are undoing the changes of the previous files. Create just the final version

Reply:
Done